airbyte-integrations/connectors/destination-redis/src/main/resources/spec.json

```diff
@@ -40,13 +40,90 @@
+        "type": "boolean",
+        "description": "Indicates whether SSL encryption protocol will be used to connect to Redis. It is recommended to use SSL connection if possible.",
+        "default": false,
+        "order": 5
+      },
+      "ssl_mode": {
+        "title": "SSL Modes",
+        "description": "SSL connection modes. \n  <li><b>verify-full</b> - This is the most secure mode. Always require encryption and verifies the identity of the source database server",
+        "type": "object",
+        "order": 6,
+        "oneOf": [
+          {
+            "title": "disable",
+            "additionalProperties": false,
+            "description": "Disable SSL.",
+            "required": ["mode"],
+            "properties": {
+              "mode": {
+                "type": "string",
+                "const": "disable",
+                "enum": ["disable"],
+                "default": "disable",
+                "order": 0
+              }
+            }
+          },
+          {
+            "title": "verify-full",
+            "additionalProperties": false,
+            "description": "Verify-full SSL mode.",
+            "required": ["mode", "ca_certificate", "client_certificate", "client_key"],
+            "properties": {
+              "mode": {
+                "type": "string",
+                "const": "verify-full",
+                "enum": ["verify-full"],
+                "default": "verify-full",
+                "order": 0
+              },
+              "ca_certificate": {
+                "type": "string",
+                "title": "CA Certificate",
+                "description": "CA certificate",
+                "airbyte_secret": true,
+                "multiline": true,
+                "order": 1
+              },
+              "client_certificate": {
+                "type": "string",
+                "title": "Client Certificate",
+                "description": "Client certificate",
+                "airbyte_secret": true,
+                "multiline": true,
+                "order": 2
+              },
+              "client_key": {
+                "type": "string",
+                "title": "Client Key",
+                "description": "Client key",
+                "airbyte_secret": true,
+                "multiline": true,
+                "order": 3
+              },
+              "client_key_password": {
+                "type": "string",
+                "title": "Client key password",
+                "description": "Password for keystorage. If you do not add it - the password will be generated automatically.",
+                "airbyte_secret": true,
+                "order": 4
+              }
+            }
+          }
+        ]
+      },
+
       "cache_type": {
         "title": "Cache type",
         "type": "string",
         "default": "hash",
         "description": "Redis cache type to store data in.",
         "enum": ["hash"],
-        "order": 5
+        "order": 6
```

Comment:
wouldn't this be `"order": 7` since `ssl.order == 5` and `ssl_mode.order == 6`?

Reply:
True